packages/client/tests/unit.utils.test.ts

```diff
@@ -38,6 +38,21 @@ describe('buildFileTree()', () => {
     );
   });
 
+  it('should include symlinked files and directories', async () => {
+    const cwd = fixture('symlinks');
+    const { fileList } = await buildFileTree(cwd, { isDirectory: true }, noop);
+
+    const expectedFileList = toAbsolutePaths(cwd, [
+      'folder-link',
+      'folder/text.txt',
+      'index.txt',
+      'index-link.txt',
+    ]);
+    expect(normalizeWindowsPaths(expectedFileList).sort()).toEqual(
```

Comment:
Should we also assert the `mode` of the file to ensure the symlink is preserved?

Reply:
added assertions